ERROR - 2022-06-28 15:43:54 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: YES)
CRITICAL - 2022-06-28 15:43:54 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
#0 C:\xampp\htdocs\mijar\system\Session\Handlers\DatabaseHandler.php(94): CodeIgniter\Database\BaseConnection->initialize()
#1 [internal function]: CodeIgniter\Session\Handlers\DatabaseHandler->open('ci_sessions', 'ci_session')
#2 C:\xampp\htdocs\mijar\system\Session\Session.php(894): session_start()
#3 C:\xampp\htdocs\mijar\system\Session\Session.php(238): CodeIgniter\Session\Session->startSession()
#4 C:\xampp\htdocs\mijar\system\Config\Services.php(577): CodeIgniter\Session\Session->start()
#5 C:\xampp\htdocs\mijar\system\Config\BaseService.php(248): CodeIgniter\Config\Services::session(Object(Config\App), false)
#6 C:\xampp\htdocs\mijar\system\Config\BaseService.php(189): CodeIgniter\Config\BaseService::__callStatic('session', Array)
#7 C:\xampp\htdocs\mijar\system\Config\Services.php(563): CodeIgniter\Config\BaseService::getSharedInstance('session', NULL)
#8 C:\xampp\htdocs\mijar\system\Config\BaseService.php(248): CodeIgniter\Config\Services::session()
#9 C:\xampp\htdocs\mijar\app\Config\Globals.php(27): CodeIgniter\Config\BaseService::__callStatic('session', Array)
#10 C:\xampp\htdocs\mijar\app\Config\Globals.php(108): Config\Globals::setGlobals()
#11 C:\xampp\htdocs\mijar\system\Autoloader\Autoloader.php(264): include_once('C:\\xampp\\htdocs...')
#12 C:\xampp\htdocs\mijar\system\Autoloader\Autoloader.php(241): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp\\htdocs...')
#13 C:\xampp\htdocs\mijar\system\Autoloader\Autoloader.php(219): CodeIgniter\Autoloader\Autoloader->loadInNamespace('Config\\Globals')
#14 C:\xampp\htdocs\mijar\app\Config\Routes.php(7): CodeIgniter\Autoloader\Autoloader->loadClass('Config\\Globals')
#15 C:\xampp\htdocs\mijar\system\CodeIgniter.php(712): require('C:\\xampp\\htdocs...')
#16 C:\xampp\htdocs\mijar\system\CodeIgniter.php(367): CodeIgniter\CodeIgniter->tryToRouteIt(Object(CodeIgniter\Router\RouteCollection))
#17 C:\xampp\htdocs\mijar\system\CodeIgniter.php(325): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#18 C:\xampp\htdocs\mijar\index.php(47): CodeIgniter\CodeIgniter->run()
#19 {main}
ERROR - 2022-06-28 15:44:22 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: YES)
CRITICAL - 2022-06-28 15:44:22 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
#0 C:\xampp\htdocs\mijar\system\Session\Handlers\DatabaseHandler.php(94): CodeIgniter\Database\BaseConnection->initialize()
#1 [internal function]: CodeIgniter\Session\Handlers\DatabaseHandler->open('ci_sessions', 'ci_session')
#2 C:\xampp\htdocs\mijar\system\Session\Session.php(894): session_start()
#3 C:\xampp\htdocs\mijar\system\Session\Session.php(238): CodeIgniter\Session\Session->startSession()
#4 C:\xampp\htdocs\mijar\system\Config\Services.php(577): CodeIgniter\Session\Session->start()
#5 C:\xampp\htdocs\mijar\system\Config\BaseService.php(248): CodeIgniter\Config\Services::session(Object(Config\App), false)
#6 C:\xampp\htdocs\mijar\system\Config\BaseService.php(189): CodeIgniter\Config\BaseService::__callStatic('session', Array)
#7 C:\xampp\htdocs\mijar\system\Config\Services.php(563): CodeIgniter\Config\BaseService::getSharedInstance('session', NULL)
#8 C:\xampp\htdocs\mijar\system\Config\BaseService.php(248): CodeIgniter\Config\Services::session()
#9 C:\xampp\htdocs\mijar\app\Config\Globals.php(27): CodeIgniter\Config\BaseService::__callStatic('session', Array)
#10 C:\xampp\htdocs\mijar\app\Config\Globals.php(108): Config\Globals::setGlobals()
#11 C:\xampp\htdocs\mijar\system\Autoloader\Autoloader.php(264): include_once('C:\\xampp\\htdocs...')
#12 C:\xampp\htdocs\mijar\system\Autoloader\Autoloader.php(241): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp\\htdocs...')
#13 C:\xampp\htdocs\mijar\system\Autoloader\Autoloader.php(219): CodeIgniter\Autoloader\Autoloader->loadInNamespace('Config\\Globals')
#14 C:\xampp\htdocs\mijar\app\Config\Routes.php(7): CodeIgniter\Autoloader\Autoloader->loadClass('Config\\Globals')
#15 C:\xampp\htdocs\mijar\system\CodeIgniter.php(712): require('C:\\xampp\\htdocs...')
#16 C:\xampp\htdocs\mijar\system\CodeIgniter.php(367): CodeIgniter\CodeIgniter->tryToRouteIt(Object(CodeIgniter\Router\RouteCollection))
#17 C:\xampp\htdocs\mijar\system\CodeIgniter.php(325): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#18 C:\xampp\htdocs\mijar\index.php(47): CodeIgniter\CodeIgniter->run()
#19 {main}
ERROR - 2022-06-28 15:46:57 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: YES)
CRITICAL - 2022-06-28 15:46:57 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
#0 C:\xampp\htdocs\mijar\system\Session\Handlers\DatabaseHandler.php(94): CodeIgniter\Database\BaseConnection->initialize()
#1 [internal function]: CodeIgniter\Session\Handlers\DatabaseHandler->open('ci_sessions', 'ci_session')
#2 C:\xampp\htdocs\mijar\system\Session\Session.php(894): session_start()
#3 C:\xampp\htdocs\mijar\system\Session\Session.php(238): CodeIgniter\Session\Session->startSession()
#4 C:\xampp\htdocs\mijar\system\Config\Services.php(577): CodeIgniter\Session\Session->start()
#5 C:\xampp\htdocs\mijar\system\Config\BaseService.php(248): CodeIgniter\Config\Services::session(Object(Config\App), false)
#6 C:\xampp\htdocs\mijar\system\Config\BaseService.php(189): CodeIgniter\Config\BaseService::__callStatic('session', Array)
#7 C:\xampp\htdocs\mijar\system\Config\Services.php(563): CodeIgniter\Config\BaseService::getSharedInstance('session', NULL)
#8 C:\xampp\htdocs\mijar\system\Config\BaseService.php(248): CodeIgniter\Config\Services::session()
#9 C:\xampp\htdocs\mijar\app\Config\Globals.php(27): CodeIgniter\Config\BaseService::__callStatic('session', Array)
#10 C:\xampp\htdocs\mijar\app\Config\Globals.php(108): Config\Globals::setGlobals()
#11 C:\xampp\htdocs\mijar\system\Autoloader\Autoloader.php(264): include_once('C:\\xampp\\htdocs...')
#12 C:\xampp\htdocs\mijar\system\Autoloader\Autoloader.php(241): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp\\htdocs...')
#13 C:\xampp\htdocs\mijar\system\Autoloader\Autoloader.php(219): CodeIgniter\Autoloader\Autoloader->loadInNamespace('Config\\Globals')
#14 C:\xampp\htdocs\mijar\app\Config\Routes.php(7): CodeIgniter\Autoloader\Autoloader->loadClass('Config\\Globals')
#15 C:\xampp\htdocs\mijar\system\CodeIgniter.php(712): require('C:\\xampp\\htdocs...')
#16 C:\xampp\htdocs\mijar\system\CodeIgniter.php(367): CodeIgniter\CodeIgniter->tryToRouteIt(Object(CodeIgniter\Router\RouteCollection))
#17 C:\xampp\htdocs\mijar\system\CodeIgniter.php(325): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#18 C:\xampp\htdocs\mijar\index.php(47): CodeIgniter\CodeIgniter->run()
#19 {main}
ERROR - 2022-06-28 15:49:26 --> Error connecting to the database: php_network_getaddresses: getaddrinfo failed: No such host is known. 
CRITICAL - 2022-06-28 15:49:26 --> Unable to connect to the database.
Main connection [MySQLi]: php_network_getaddresses: getaddrinfo failed: No such host is known. 
#0 C:\xampp\htdocs\mijar\system\Session\Handlers\DatabaseHandler.php(94): CodeIgniter\Database\BaseConnection->initialize()
#1 [internal function]: CodeIgniter\Session\Handlers\DatabaseHandler->open('ci_sessions', 'ci_session')
#2 C:\xampp\htdocs\mijar\system\Session\Session.php(894): session_start()
#3 C:\xampp\htdocs\mijar\system\Session\Session.php(238): CodeIgniter\Session\Session->startSession()
#4 C:\xampp\htdocs\mijar\system\Config\Services.php(577): CodeIgniter\Session\Session->start()
#5 C:\xampp\htdocs\mijar\system\Config\BaseService.php(248): CodeIgniter\Config\Services::session(Object(Config\App), false)
#6 C:\xampp\htdocs\mijar\system\Config\BaseService.php(189): CodeIgniter\Config\BaseService::__callStatic('session', Array)
#7 C:\xampp\htdocs\mijar\system\Config\Services.php(563): CodeIgniter\Config\BaseService::getSharedInstance('session', NULL)
#8 C:\xampp\htdocs\mijar\system\Config\BaseService.php(248): CodeIgniter\Config\Services::session()
#9 C:\xampp\htdocs\mijar\app\Config\Globals.php(27): CodeIgniter\Config\BaseService::__callStatic('session', Array)
#10 C:\xampp\htdocs\mijar\app\Config\Globals.php(108): Config\Globals::setGlobals()
#11 C:\xampp\htdocs\mijar\system\Autoloader\Autoloader.php(264): include_once('C:\\xampp\\htdocs...')
#12 C:\xampp\htdocs\mijar\system\Autoloader\Autoloader.php(241): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp\\htdocs...')
#13 C:\xampp\htdocs\mijar\system\Autoloader\Autoloader.php(219): CodeIgniter\Autoloader\Autoloader->loadInNamespace('Config\\Globals')
#14 C:\xampp\htdocs\mijar\app\Config\Routes.php(7): CodeIgniter\Autoloader\Autoloader->loadClass('Config\\Globals')
#15 C:\xampp\htdocs\mijar\system\CodeIgniter.php(712): require('C:\\xampp\\htdocs...')
#16 C:\xampp\htdocs\mijar\system\CodeIgniter.php(367): CodeIgniter\CodeIgniter->tryToRouteIt(Object(CodeIgniter\Router\RouteCollection))
#17 C:\xampp\htdocs\mijar\system\CodeIgniter.php(325): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#18 C:\xampp\htdocs\mijar\index.php(47): CodeIgniter\CodeIgniter->run()
#19 {main}
ERROR - 2022-06-28 15:56:13 --> Error connecting to the database: php_network_getaddresses: getaddrinfo failed: No such host is known. 
CRITICAL - 2022-06-28 15:56:13 --> Unable to connect to the database.
Main connection [MySQLi]: php_network_getaddresses: getaddrinfo failed: No such host is known. 
#0 C:\xampp\htdocs\mijar\system\Session\Handlers\DatabaseHandler.php(94): CodeIgniter\Database\BaseConnection->initialize()
#1 [internal function]: CodeIgniter\Session\Handlers\DatabaseHandler->open('ci_sessions', 'ci_session')
#2 C:\xampp\htdocs\mijar\system\Session\Session.php(894): session_start()
#3 C:\xampp\htdocs\mijar\system\Session\Session.php(238): CodeIgniter\Session\Session->startSession()
#4 C:\xampp\htdocs\mijar\system\Config\Services.php(577): CodeIgniter\Session\Session->start()
#5 C:\xampp\htdocs\mijar\system\Config\BaseService.php(248): CodeIgniter\Config\Services::session(Object(Config\App), false)
#6 C:\xampp\htdocs\mijar\system\Config\BaseService.php(189): CodeIgniter\Config\BaseService::__callStatic('session', Array)
#7 C:\xampp\htdocs\mijar\system\Config\Services.php(563): CodeIgniter\Config\BaseService::getSharedInstance('session', NULL)
#8 C:\xampp\htdocs\mijar\system\Config\BaseService.php(248): CodeIgniter\Config\Services::session()
#9 C:\xampp\htdocs\mijar\app\Config\Globals.php(27): CodeIgniter\Config\BaseService::__callStatic('session', Array)
#10 C:\xampp\htdocs\mijar\app\Config\Globals.php(108): Config\Globals::setGlobals()
#11 C:\xampp\htdocs\mijar\system\Autoloader\Autoloader.php(264): include_once('C:\\xampp\\htdocs...')
#12 C:\xampp\htdocs\mijar\system\Autoloader\Autoloader.php(241): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp\\htdocs...')
#13 C:\xampp\htdocs\mijar\system\Autoloader\Autoloader.php(219): CodeIgniter\Autoloader\Autoloader->loadInNamespace('Config\\Globals')
#14 C:\xampp\htdocs\mijar\app\Config\Routes.php(7): CodeIgniter\Autoloader\Autoloader->loadClass('Config\\Globals')
#15 C:\xampp\htdocs\mijar\system\CodeIgniter.php(712): require('C:\\xampp\\htdocs...')
#16 C:\xampp\htdocs\mijar\system\CodeIgniter.php(367): CodeIgniter\CodeIgniter->tryToRouteIt(Object(CodeIgniter\Router\RouteCollection))
#17 C:\xampp\htdocs\mijar\system\CodeIgniter.php(325): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#18 C:\xampp\htdocs\mijar\index.php(47): CodeIgniter\CodeIgniter->run()
#19 {main}
ERROR - 2022-06-28 15:56:23 --> Error connecting to the database: php_network_getaddresses: getaddrinfo failed: No such host is known. 
CRITICAL - 2022-06-28 15:56:23 --> Unable to connect to the database.
Main connection [MySQLi]: php_network_getaddresses: getaddrinfo failed: No such host is known. 
#0 C:\xampp\htdocs\mijar\system\Session\Handlers\DatabaseHandler.php(94): CodeIgniter\Database\BaseConnection->initialize()
#1 [internal function]: CodeIgniter\Session\Handlers\DatabaseHandler->open('ci_sessions', 'ci_session')
#2 C:\xampp\htdocs\mijar\system\Session\Session.php(894): session_start()
#3 C:\xampp\htdocs\mijar\system\Session\Session.php(238): CodeIgniter\Session\Session->startSession()
#4 C:\xampp\htdocs\mijar\system\Config\Services.php(577): CodeIgniter\Session\Session->start()
#5 C:\xampp\htdocs\mijar\system\Config\BaseService.php(248): CodeIgniter\Config\Services::session(Object(Config\App), false)
#6 C:\xampp\htdocs\mijar\system\Config\BaseService.php(189): CodeIgniter\Config\BaseService::__callStatic('session', Array)
#7 C:\xampp\htdocs\mijar\system\Config\Services.php(563): CodeIgniter\Config\BaseService::getSharedInstance('session', NULL)
#8 C:\xampp\htdocs\mijar\system\Config\BaseService.php(248): CodeIgniter\Config\Services::session()
#9 C:\xampp\htdocs\mijar\app\Config\Globals.php(27): CodeIgniter\Config\BaseService::__callStatic('session', Array)
#10 C:\xampp\htdocs\mijar\app\Config\Globals.php(108): Config\Globals::setGlobals()
#11 C:\xampp\htdocs\mijar\system\Autoloader\Autoloader.php(264): include_once('C:\\xampp\\htdocs...')
#12 C:\xampp\htdocs\mijar\system\Autoloader\Autoloader.php(241): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp\\htdocs...')
#13 C:\xampp\htdocs\mijar\system\Autoloader\Autoloader.php(219): CodeIgniter\Autoloader\Autoloader->loadInNamespace('Config\\Globals')
#14 C:\xampp\htdocs\mijar\app\Config\Routes.php(7): CodeIgniter\Autoloader\Autoloader->loadClass('Config\\Globals')
#15 C:\xampp\htdocs\mijar\system\CodeIgniter.php(712): require('C:\\xampp\\htdocs...')
#16 C:\xampp\htdocs\mijar\system\CodeIgniter.php(367): CodeIgniter\CodeIgniter->tryToRouteIt(Object(CodeIgniter\Router\RouteCollection))
#17 C:\xampp\htdocs\mijar\system\CodeIgniter.php(325): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#18 C:\xampp\htdocs\mijar\index.php(47): CodeIgniter\CodeIgniter->run()
#19 {main}
ERROR - 2022-06-28 15:56:28 --> Error connecting to the database: php_network_getaddresses: getaddrinfo failed: No such host is known. 
CRITICAL - 2022-06-28 15:56:28 --> Unable to connect to the database.
Main connection [MySQLi]: php_network_getaddresses: getaddrinfo failed: No such host is known. 
#0 C:\xampp\htdocs\mijar\system\Session\Handlers\DatabaseHandler.php(94): CodeIgniter\Database\BaseConnection->initialize()
#1 [internal function]: CodeIgniter\Session\Handlers\DatabaseHandler->open('ci_sessions', 'ci_session')
#2 C:\xampp\htdocs\mijar\system\Session\Session.php(894): session_start()
#3 C:\xampp\htdocs\mijar\system\Session\Session.php(238): CodeIgniter\Session\Session->startSession()
#4 C:\xampp\htdocs\mijar\system\Config\Services.php(577): CodeIgniter\Session\Session->start()
#5 C:\xampp\htdocs\mijar\system\Config\BaseService.php(248): CodeIgniter\Config\Services::session(Object(Config\App), false)
#6 C:\xampp\htdocs\mijar\system\Config\BaseService.php(189): CodeIgniter\Config\BaseService::__callStatic('session', Array)
#7 C:\xampp\htdocs\mijar\system\Config\Services.php(563): CodeIgniter\Config\BaseService::getSharedInstance('session', NULL)
#8 C:\xampp\htdocs\mijar\system\Config\BaseService.php(248): CodeIgniter\Config\Services::session()
#9 C:\xampp\htdocs\mijar\app\Config\Globals.php(27): CodeIgniter\Config\BaseService::__callStatic('session', Array)
#10 C:\xampp\htdocs\mijar\app\Config\Globals.php(108): Config\Globals::setGlobals()
#11 C:\xampp\htdocs\mijar\system\Autoloader\Autoloader.php(264): include_once('C:\\xampp\\htdocs...')
#12 C:\xampp\htdocs\mijar\system\Autoloader\Autoloader.php(241): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp\\htdocs...')
#13 C:\xampp\htdocs\mijar\system\Autoloader\Autoloader.php(219): CodeIgniter\Autoloader\Autoloader->loadInNamespace('Config\\Globals')
#14 C:\xampp\htdocs\mijar\app\Config\Routes.php(7): CodeIgniter\Autoloader\Autoloader->loadClass('Config\\Globals')
#15 C:\xampp\htdocs\mijar\system\CodeIgniter.php(712): require('C:\\xampp\\htdocs...')
#16 C:\xampp\htdocs\mijar\system\CodeIgniter.php(367): CodeIgniter\CodeIgniter->tryToRouteIt(Object(CodeIgniter\Router\RouteCollection))
#17 C:\xampp\htdocs\mijar\system\CodeIgniter.php(325): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#18 C:\xampp\htdocs\mijar\index.php(47): CodeIgniter\CodeIgniter->run()
#19 {main}
ERROR - 2022-06-28 15:58:42 --> Error connecting to the database: php_network_getaddresses: getaddrinfo failed: No such host is known. 
CRITICAL - 2022-06-28 15:58:42 --> Unable to connect to the database.
Main connection [MySQLi]: php_network_getaddresses: getaddrinfo failed: No such host is known. 
#0 C:\xampp\htdocs\mijar\system\Session\Handlers\DatabaseHandler.php(94): CodeIgniter\Database\BaseConnection->initialize()
#1 [internal function]: CodeIgniter\Session\Handlers\DatabaseHandler->open('ci_sessions', 'ci_session')
#2 C:\xampp\htdocs\mijar\system\Session\Session.php(894): session_start()
#3 C:\xampp\htdocs\mijar\system\Session\Session.php(238): CodeIgniter\Session\Session->startSession()
#4 C:\xampp\htdocs\mijar\system\Config\Services.php(577): CodeIgniter\Session\Session->start()
#5 C:\xampp\htdocs\mijar\system\Config\BaseService.php(248): CodeIgniter\Config\Services::session(Object(Config\App), false)
#6 C:\xampp\htdocs\mijar\system\Config\BaseService.php(189): CodeIgniter\Config\BaseService::__callStatic('session', Array)
#7 C:\xampp\htdocs\mijar\system\Config\Services.php(563): CodeIgniter\Config\BaseService::getSharedInstance('session', NULL)
#8 C:\xampp\htdocs\mijar\system\Config\BaseService.php(248): CodeIgniter\Config\Services::session()
#9 C:\xampp\htdocs\mijar\app\Config\Globals.php(27): CodeIgniter\Config\BaseService::__callStatic('session', Array)
#10 C:\xampp\htdocs\mijar\app\Config\Globals.php(108): Config\Globals::setGlobals()
#11 C:\xampp\htdocs\mijar\system\Autoloader\Autoloader.php(264): include_once('C:\\xampp\\htdocs...')
#12 C:\xampp\htdocs\mijar\system\Autoloader\Autoloader.php(241): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp\\htdocs...')
#13 C:\xampp\htdocs\mijar\system\Autoloader\Autoloader.php(219): CodeIgniter\Autoloader\Autoloader->loadInNamespace('Config\\Globals')
#14 C:\xampp\htdocs\mijar\app\Config\Routes.php(7): CodeIgniter\Autoloader\Autoloader->loadClass('Config\\Globals')
#15 C:\xampp\htdocs\mijar\system\CodeIgniter.php(712): require('C:\\xampp\\htdocs...')
#16 C:\xampp\htdocs\mijar\system\CodeIgniter.php(367): CodeIgniter\CodeIgniter->tryToRouteIt(Object(CodeIgniter\Router\RouteCollection))
#17 C:\xampp\htdocs\mijar\system\CodeIgniter.php(325): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#18 C:\xampp\htdocs\mijar\index.php(47): CodeIgniter\CodeIgniter->run()
#19 {main}
ERROR - 2022-06-28 16:04:30 --> Error connecting to the database: php_network_getaddresses: getaddrinfo failed: No such host is known. 
CRITICAL - 2022-06-28 16:04:30 --> Unable to connect to the database.
Main connection [MySQLi]: php_network_getaddresses: getaddrinfo failed: No such host is known. 
#0 C:\xampp\htdocs\mijar\system\Session\Handlers\DatabaseHandler.php(94): CodeIgniter\Database\BaseConnection->initialize()
#1 [internal function]: CodeIgniter\Session\Handlers\DatabaseHandler->open('ci_sessions', 'ci_session')
#2 C:\xampp\htdocs\mijar\system\Session\Session.php(894): session_start()
#3 C:\xampp\htdocs\mijar\system\Session\Session.php(238): CodeIgniter\Session\Session->startSession()
#4 C:\xampp\htdocs\mijar\system\Config\Services.php(577): CodeIgniter\Session\Session->start()
#5 C:\xampp\htdocs\mijar\system\Config\BaseService.php(248): CodeIgniter\Config\Services::session(Object(Config\App), false)
#6 C:\xampp\htdocs\mijar\system\Config\BaseService.php(189): CodeIgniter\Config\BaseService::__callStatic('session', Array)
#7 C:\xampp\htdocs\mijar\system\Config\Services.php(563): CodeIgniter\Config\BaseService::getSharedInstance('session', NULL)
#8 C:\xampp\htdocs\mijar\system\Config\BaseService.php(248): CodeIgniter\Config\Services::session()
#9 C:\xampp\htdocs\mijar\app\Config\Globals.php(27): CodeIgniter\Config\BaseService::__callStatic('session', Array)
#10 C:\xampp\htdocs\mijar\app\Config\Globals.php(108): Config\Globals::setGlobals()
#11 C:\xampp\htdocs\mijar\system\Autoloader\Autoloader.php(264): include_once('C:\\xampp\\htdocs...')
#12 C:\xampp\htdocs\mijar\system\Autoloader\Autoloader.php(241): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp\\htdocs...')
#13 C:\xampp\htdocs\mijar\system\Autoloader\Autoloader.php(219): CodeIgniter\Autoloader\Autoloader->loadInNamespace('Config\\Globals')
#14 C:\xampp\htdocs\mijar\app\Config\Routes.php(7): CodeIgniter\Autoloader\Autoloader->loadClass('Config\\Globals')
#15 C:\xampp\htdocs\mijar\system\CodeIgniter.php(712): require('C:\\xampp\\htdocs...')
#16 C:\xampp\htdocs\mijar\system\CodeIgniter.php(367): CodeIgniter\CodeIgniter->tryToRouteIt(Object(CodeIgniter\Router\RouteCollection))
#17 C:\xampp\htdocs\mijar\system\CodeIgniter.php(325): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#18 C:\xampp\htdocs\mijar\index.php(47): CodeIgniter\CodeIgniter->run()
#19 {main}
